lib/modules/auphonic/admin.js

```diff
@@ -261,8 +261,9 @@ var PODLOVE = PODLOVE || {};
 
 			data.length_timestring = $("#_podlove_meta_duration").val();
 			data.output_basename = $("#_podlove_meta_slug").val();
-			data.metadata.title = $("#title").val();
+			data.metadata.title = $("#_podlove_meta_title").val() || $("#title").val();
 			data.metadata.subtitle = $("#_podlove_meta_subtitle").val();
+			data.metadata.track = $("#row__podlove_meta_number").val();
```

Comment:
Looking at the [Auphonic API docs](https://auphonic.com/help/api/details.html#create-a-production-with-detailed-audio-metadata), this seems to be supposed to be a number, maybe need to cast it? Not sure and wasn't able to test this yet myself.

Reply:
Would be good to ensure this is actually numeric, yes.

Reply:
Sure, done!